modules/local/metabinner/main.nf

```diff
@@ -0,0 +1,76 @@
+process METABINNER {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/metabinner:1.4.4--hdfd78af_0' :
+        'quay.io/biocontainers/metabinner:1.4.4--hdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(fasta), path(depth)
+
+    output:
+    tuple val(meta), path("*.tooShort.fa.gz")         , emit: tooshort
+    tuple val(meta), path("*.lowDepth.fa.gz")         , optional:true , emit: lowdepth
+    tuple val(meta), path("*.unbinned.fa.gz")         , emit: unbinned
+    tuple val(meta), path("*.tsv.gz")                 , emit: membership
+    tuple val(meta), path("bins/*.fa.gz")             , emit: bins
+    tuple val(meta), path("*.log.gz")                 , emit: log
+    tuple val(meta), path("coverage_profile.tsv")     , emit: coverage_profile
+    tuple val(meta), path("*.csv.gz")                 , emit: composition_profile
+    path "versions.yml"                               , emit: versions
+
+    script:
+    def args   = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def min_contig_size = task.ext.min_contig_size ?: "1000"
+    """
+    metabinner_path=\$(dirname \$(which run_metabinner.sh))
```

Comment:
Do these scripts not get put on the path? ...😓 Should really have a conda recipe fix